Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
Sincerely I am not sure if there's a recommendation or directive to host all the Docker related information into a `./docker` directory. Maybe @jmcanterafonseca will able to solve it for us... :)

If this is the case, the 'config-docker.js' file should also be moved to this directory and the `Dockerfile` should be updated accordingly.

BTW, I just noticed that the changes which have to be included into the `Dockerfile` (to adapt it to this new `./docker` directory) probably suggest keeping the Docker related information in the root of the repository :) I mean regarding the paths to be used in the 'Dockerfile' and the path from where the `docker xxx` commands may be invoked from :) What do you think guys?